tests/swap_test.go

```diff
@@ -224,7 +224,7 @@ var _ = Describe("[Serial][sig-compute]SwapTest", Serial, decorators.SigCompute,
 })
 
 func getMemInfoByString(node v1.Node, field string) int {
-	stdout, stderr, err := tests.ExecuteCommandOnNodeThroughVirtHandler(kubevirt.Client(), node.Name, []string{"grep", field, "/proc/meminfo"})
+	stdout, stderr, err := libnode.ExecuteCommandOnNodeThroughVirtHandler(kubevirt.Client(), node.Name, []string{"grep", field, "/proc/meminfo"})
```

Comment:
`kubevirt.Client()` can be used in the function itself.

Reply:
It is not related to the PR scope (appears on PR title and desc), 
lets focus please on the PR scope
adding it to another commit can be even confusing

Reply:
It's up to you, but if you already adding new changes, I think you still can remove it from the function signature.